### PR TITLE
Add unit test for simulation renderer with colors

### PR DESCRIPTION
**Pull request by karjonas**
Move test files into separate folders.

Fix bug where animation frame was always set to zero when end frame was updated.

### CODE REVIEW

**Comment by karjonas** [comment]
retest this please

**Comment by ppodhajski** [comment]
retest this please

File: tests/paths.h.in [hunk 1]
```diff
@@ -9,9 +9,20 @@
...
 
-static char const *const BRAYNS_TESTDATA = "${CMAKE_CURRENT_SOURCE_DIR}/";
+#define BRAYNS_TESTDATA_PATH "${CMAKE_CURRENT_SOURCE_DIR}/files/"
 
+#define BRAYNS_TESTDATA_IMAGES_PATH BRAYNS_TESTDATA_PATH "images/"
+#define BRAYNS_TESTDATA_VALID_MODELS_PATH BRAYNS_TESTDATA_PATH "modelsValid/"
+
+constexpr auto BRAYNS_TESTDATA_MODEL_BENNU_PATH =
+    BRAYNS_TESTDATA_PATH "modelsValid/bennu.obj";
+constexpr auto BRYNAS_TESTDATA_MODEL_MONKEY_PATH =
```
Comment (tribal-tec) [inline tests/paths.h.in]: what the hell is BRYNAS?

**Comment by ppodhajski** [comment]
retest this please

**Comment by ppodhajski** [comment]
retest this please

**Comment by ppodhajski** [comment]
retest this please

**Comment by ppodhajski** [comment]
retest this please

**Comment by karjonas** [comment]
retest this please

**Pull request merged**